src/indicator/git.ts

```diff
@@ -15,20 +15,20 @@ function fromGitBoolean(data: string): boolean {
     data = data.toLowerCase();
     let result: boolean = false;
     switch (data) {
-    case 'true':
-    case 'yes':
-    case 'on':
-    case '1':
-        result = true;
-        break;
+        case 'true':
+        case 'yes':
+        case 'on':
+        case '1':
+            result = true;
+            break;
```

Comment:
The original indention style was adhere to default ESLint config.
I thought we should not modify this part? :D

Reply:
Well spotted, I use VSCode default `format-on-save` settings so that's why this got reformatted that way :) not intentionally I got to say. I realise now my current config is not ESLint complaint :D

Reply:
Ok, I got it. :D

May you modify this and force push to your master branch again?

It's a little late in my country, I will merge this and push next version onto marketplace tomorrow.

Reply:
Definitively, sorry I was having lunch! yes, makes sense. I'll update my fork and it should get reflected here.

Cheers!

Reply:
Changes were reverted as per this conversation